src/render-html/html.js

```diff
@@ -14,7 +14,7 @@ ${script(initOptions.anchor)}
 ${css(theme, initOptions.highlightUnreadMessages)}
 
 <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
-<body>
+<body style="overflow-x: hidden;">
```

Comment:
On older devices it takes some time to load/apply css.
(momentary flash doesn't look good)